### ISSUE TITLE
1.17

### ISSUE THREAD

**Issue by CCF100** [dzwdz/chat_heads#11]

**Comment by Gontier-Julien** [comment]
I've made a pull request to the dev, but if you want it now check my [fork](https://github.com/Gontier-Julien/chat_heads) where i've already build chat_head for the 1.17

**Comment by dzwdz** [comment]
yup, i'd just recommend you to use their fork for the time being - I want to update all my mods to 1.17, but I'm pretty busy recently and I just haven't found the time to do it yet.

**Comment by Gontier-Julien** [comment]
It now updated to 1.17.1 now on my fork

**Comment by AlisterGontineac** [comment]
So weird, I downloaded chat_heads-0.3+1.17 on august 21st. Did it get taken down?
